lambdas/process.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const https = require('https');
+const AWS = require('aws-sdk');
+
+const targetBucket = process.env.S3_BUCKET; // receiver bucket name
+const s3 = new AWS.S3();
```

Comment:
I'm not familiar with Lambda execution model, but I presume it's safe to declare variable here instead of inside the function because there's no concurrency, right?

Reply:
yes.